images/cassandra-recovery/pkg/sts/statefulset.go

```diff
@@ -0,0 +1,224 @@
+package sts
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+var evictionLabel = os.Getenv("EVICTION_LABEL")
```

Comment:
What happens if this isn't set? It would be great if the env var would be checked in `main.go` and StatefulSet processing could be configured with a string instead of an env var.

Reply:
I've reworked that. I initialize this now in the `controller.go` (simply because all other env vars are read there as well) and log a startup message.

If no eviction label is set, the controller still starts but doesn't support eviction.